src/components/Header.js

```diff
@@ -1,19 +1,7 @@
-import Link from 'next/link';
-import {useRouter} from 'next/router';
-
 export default function Header() {
-	const router = useRouter();
 	return (
 		<header>
-			<nav>
-				<Link href="/">
-					<a>Home</a>
-				</Link>
-				<Link href="/routine">
-					<a>Routine</a>
-				</Link>
-			</nav>
-			<button onClick={() => router.back()}>zurück</button>
+			<h2>My Yoga Asana</h2>
```

Comment:
Is this a placeholder for a logo? If you only have a headline here, you do not need a `header` element as a `header` is a grouping element. If you want to keep the headline, this should be an `h1` and your current `h1` (e.g 'Asanas' in the `index.js`) should become an `h2`.

Reply:
Der Header ist absichtlich h2, damit die Semantik meiner Seiten nicht gestört sind, da ich bereits h1 nutze. In Zukunft soll mein Header ein Logo, sowie den Seiten titel halten. Ich habe den Seitentitel zu einer Span geändert ...